src/plot_erpimage.jl

```diff
@@ -52,8 +52,6 @@ plot_erpimage!(
 plot_erpimage(times::AbstractVector, data::Matrix{<:Real}; kwargs...) =
```

Comment:
maybe add here the Union(Observable) as well? Else you can only use observables with plot_erpimage!()

Reply:
I didnt see the docstring for the observables for erp_image - but maybe I missed it and they already exist. Can you confirm some user-facing documentation  exist for observables there?